src/datasources/results/components/editors/query-results/QueryResultsEditor.tsx

```diff
@@ -148,11 +155,11 @@ export function UseTimeRangeControls({ query, handleQueryChange }: Props) {
 }
 
 const tooltips = {
-  output: 'Select the output type for the query',
-  properties: 'Select the properties fields to query',
-  recordCount: 'Enter the number of records to query',
-  orderBy: 'Select the field to order the results by',
-  descending: 'Select to order the results in descending order',
-  useTimeRange: 'Select to query using the dashboard time range for the selected field',
-  useTimeRangeFor: 'Select the field to apply the dashboard time range',
+  output: 'Specifies the type of output for the queried result.',
+  properties: 'Specifies the properties to be queried.',
+  recordCount: 'Specifies the maximum number of results to return.',
+  orderBy: 'Specifies the field to order the queried results by.',
+  descending: 'Specifies whether to return the queried results in descending order.',
+  useTimeRange: 'Specifies whether to query only for data within the dashboard time range for the selected property.',
```

Comment:
@sudharsan-ni @karthikb-ni - These are the tooltips for the respective inputs. `useTimeRange` and `useTimeRangeFor` are the new fields added - all other fields follow similar tooltips as [products](https://github.com/ni/systemlink-grafana-plugins/blob/main/src/datasources/products/components/ProductsQueryEditor.tsx#L147-L151)

Reply:
```suggestion
  useTimeRange: 'Only query data within the dashboard's time range.',
```

@sudharsan-ni - I would want us to move away from the "Specifies" prefix added to each of the properties, try to be more precise and explain what would happen for the specific control interaction - your thoughts?

Reply:
I would agree with your suggestion @ranjgith-ni

Reply:
@sudharsan-ni i have pinged Will to see if the tech writers even deal with Grafana strings or not because none of us were sure of that. We are not even sure if they have github / grafana-repo access.

While i wait for Will's response, can you please share your thoughts on the strings ?

Because if the tech writers are not involved in Grafana, then as the PO your call will be final here.

Reply:
@WillCawley Attaching screenshot showing where this tooltip will be displayed. 

1. The below tooltip will be displayed when user hovers over `output` label
![image](https://github.com/user-attachments/assets/fe75e796-b8bb-433e-a810-400f32992ea0)

2. The below tooltip will be displayed when user hovers over `properties` label
![image](https://github.com/user-attachments/assets/b4b438e3-2286-4a67-b0f8-c65fa68b10c5)

3. The below tooltip will be displayed when user hovers over `OrderBy` label
![image](https://github.com/user-attachments/assets/be45d8fc-b7be-4363-bdba-75858aeb8858)

4. The below tooltip will be displayed when user hovers over `Descending` label
![image](https://github.com/user-attachments/assets/2d83f345-0778-45d1-b30d-dcc2a6578afa)

5. The below tooltip will be displayed when user hovers over `Take` label
![image](https://github.com/user-attachments/assets/29e99ac8-1c8e-4d87-935b-acb092012384)

6. The below tooltip will be displayed when user hovers over `Use Time range` label
![image](https://github.com/user-attachments/assets/6161aef2-b347-42e3-b24a-b2ddbe6f0a3f)

7. The below tooltip will be displayed when user hovers over `to filter by` label
![image](https://github.com/user-attachments/assets/b4da1977-8007-4b8b-a3c2-861a3a223cbf)

Additionally I have added the below screenshot of results query editor to provide the overview of various label and its corresponding fields
![image](https://github.com/user-attachments/assets/bdc786c1-d899-4cfb-bb71-cad537963773)

Reply:
Hey Everyone! Thanks for giving me access. If we want consistency, we could try calling out the interactive unit in the tooltip. That would turn these sentence fragments into full sentences.

Here are some suggestions:

1.	This field specifies the output type for the query result.
2.	This field specifies the properties to use in the query.
3.	This field orders the query results by field.
4.	This toggle returns the query results in descending order.
5.	This field sets the maximum number of results.
6.	This toggle enables querying within the dashboard time range.
7.	This field specifies the property to query within the dashboard time range.

Reply:
Updated tooltips

Reply:
@Ahalya-ni Minor improvements on top of what Will suggested, just to be a bit explicit on the purpose of the field. We can go with how Will has suggested framing these sentences.

1. This field specifies the output type for the query result.
2. This field specifies the properties _to be queried_.
3. This field specifies _the field by which the query results are ordered by_.
4. This field returns the query results in descending order.
5. This field sets the maximum number of results _to be queried_.
6. This toggle enables querying within the _dashboard's global time range selection_.
7. This field specifies the property to query by within the _dashboard's global time range selection_.

I see this PR is already merged. You can update the above as part of one of the upcoming PRs.